src/test/regress/expected/multi_verify_no_subquery.out

```diff
@@ -1,18 +0,0 @@
---
--- MULTI_VERIFY_NO_SUBQUERY
```

Comment:
I don't think so. But, the tests were here to make sure that we error out on whenever we got subqueries in where clause. 
Now that the ones that we don't support would error out saying they are not joined on the partition key. And, we have a good coverage on the other test file for this.